Dashboard/app/js/lib/views/data/monitoring-data-table-view.js

```diff
@@ -109,6 +109,7 @@ FLOW.DataPointView = FLOW.View.extend({
     approvalStatus: [{label: Ember.String.loc('_pending'), value: 'PENDING'}, { label: Ember.String.loc('_approved'), value: 'APPROVED' },{ label: Ember.String.loc('_rejected'), value: 'REJECTED'}],
 
     showDataApprovalBlock: false,
+    count: 0,
```

Comment:
This var is never used anywhere. Did you perhaps leave it in by accident?

Reply:
i forgot to delete it when i was removing the custom handlebar helper part